extensions/amp-pan-zoom/amp-pan-zoom.md

```diff
@@ -30,7 +31,7 @@ Provides zooming and panning for arbitrary content.
 <table>
   <tr>
     <td width="40%"><strong>Availability</strong></td>
-    <td><div><a href="https://www.ampproject.org/docs/reference/experimental.html">Experimental</a></td>
+    <td><div><a href="https://amp.dev/documentation/guides-and-tutorials/learn/experimental">Experimental</a></td>
```

Comment:
not experimental. Docs were wrong.

Reply:
(I have updated the file in this PR myself)

Reply:
Thanks, didn't know that.